check_setup.py

```diff
@@ -13,25 +13,69 @@ def check_for_file(file_name, not_found_message):
     print("Did not find output directoty, creating one")
     os.mkdir("output")
 
-# Check for the notebook
-check_for_file("GrowthViz.ipynb", "Unable to find the GrowthViz Jupyter Notebook.")
+# Check for the notebooks
+check_for_file(
+    "GrowthViz-pediatrics.ipynb",
+    "Unable to find the GrowthViz Pediatrics Jupyter Notebook.",
+)
+check_for_file(
+    "GrowthViz-adults.ipynb", "Unable to find the GrowthViz Adults Jupyter Notebook."
+)
 
 # Check for the function library
-check_for_file("charts.py", "Unable to find library functions for GrowthViz.")
+check_for_file(
+    "processdata.py", "Unable to find data processing library functions for GrowthViz."
+)
+check_for_file(
+    "sumstats.py", "Unable to find summary statistics library functions for GrowthViz."
+)
+check_for_file(
+    "charts.py", "Unable to find visualization library functions for GrowthViz."
+)
+check_for_file(
+    "compare.py", "Unable to find comparison library functions for GrowthViz."
+)
+check_for_file(
+    "check_data.py", "Unable to find data checks library functions for GrowthViz."
+)
+
+# Check for tests
+check_for_file(
```

Comment:
Probably don't need to check for tests - these are dev-oriented, whereas the other files are user-oriented checks.

Reply:
Noted in #34, which I can take later.